jenkins-slaves/ansible/roles/common/tasks/main.yml

```
@@ -155,6 +155,35 @@
   with_items:
     - "{{ maven_versions }}"
 
+### Takari extensions for parallel builds
+-
+- name: Download takari-smart-builder
+  get_url:
+    url: http://repo1.maven.org/maven2/io/takari/maven/takari-smart-builder/0.4.0/takari-smart-builder-0.4.0.jar
```

there needs to be https instead of http
```suggestion
    url: https://repo1.maven.org/maven2/io/takari/maven/takari-smart-builder/0.4.0/takari-smart-builder-0.4.0.jar
```

---

fixed